### PR TITLE
added like to review

### PR DESCRIPTION
**Pull request by illyadmytrenko**

### CODE REVIEW

File: src/common/components/sort-dropdown/index.tsx [hunk 1]
```diff
@@ -29,7 +29,6 @@ const SortDropdown: React.FC<Properties> = ({
 	isDisabled,
 	isIconButton = false,
 	menuStaticStyle,
-	name,
```
Comment (KirillSerg) [inline src/common/components/sort-dropdown/index.tsx:32]: it is necessary, do not delete it. PR @ValentinaTenus  or mine will have a fix

File: src/pages/company-details-page/components/company-details/components/reviews/components/review-card.tsx [hunk 2]
```diff
@@ -115,11 +164,16 @@ const ReviewCard: React.FC<{
...
-						</span>
+					<Link
+						className={styles["review_icon-text"]}
+						onClick={handleLike}
+						to="#"
+					>
+						<Icon
+							className={`${styles["review_like"]} ${isLiked ? styles["liked"] : ""}`}
+							name={IconName.LIKE}
+						/>
```
Comment (ValentinaTenus) [inline src/pages/company-details-page/components/company-details/components/reviews/components/review-card.tsx:175]: why do you use Link here but not Button?

File: src/pages/company-details-page/components/company-details/components/reviews/components/statistics.tsx [hunk 3]
```diff
@@ -29,13 +35,38 @@ const Statistics: React.FC<{
...
 
 	const handleCloseReviewModal = () => {
 		setIsReviewModalOpen(false);
 	};
 
+	const [isButtonInactive, setIsButtonInactive] = useState(false);
+
+	const userCompanyReviews = JSON.parse(
+		localStorage.getItem("userCompanyReviews") || "[]",
+	);
```
Comment (ValentinaTenus) [inline src/pages/company-details-page/components/company-details/components/reviews/components/statistics.tsx:50]: use data from api

**Review by ValentinaTenus — CHANGES_REQUESTED** [review]

File: src/pages/company-details-page/components/company-details/components/reviews/components/components/review-modal.tsx [hunk 4]
```diff
@@ -44,58 +53,77 @@ const ReviewModal: React.FC<{
...
 			companyId: company.id,
 			rating: rating,
 			text: reviewText,
 		});
+		if (response.error) {
+			if (
+				"status" in response.error &&
+				response.error.status !== HttpStatusCode.FORBIDDEN
+			) {
+				window.location.href = AppRoute.PRIVACY_POLICY;
```
Comment (ValentinaTenus) [inline src/pages/company-details-page/components/company-details/components/reviews/components/components/review-modal.tsx:66]: you can use useNavigate hook instead

**Review by ValentinaTenus — APPROVED** [review]

**Review by ValentinaTenus — APPROVED** [review]

**Pull request merged**